libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1166,6 +1166,13 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info2[] = {
     AP_SUBGROUPINFO(rc_lq, "RC_LQ", 9, AP_OSD_Screen, AP_OSD_Setting),
 #endif
 
+#if HAL_WITH_ESC_TELEM
+    // @Param: ESC_IDX
+    // @DisplayName: ESC_IDX
+    // @Description: Index of the ESC to use for displaying ESC information. 0 means use the most active ESC.
```

Comment:
I'm not a fan of "most active". What does it mean? Maybe just "highest RPM".

Reply:
Fixed, plus the other bad bug you didn't spot ;)